database/database.py

```diff
@@ -0,0 +1,157 @@
+import sqlite3
+
+def connect_database():
+    conn = sqlite3.connect('database.db')
+    print("Database opened successfully")
+    return conn
+
+'''
+    Oxygen (Saturation): A Percentage
+        e.g 97% or 0.97
+    Pulse: Beats per minute
+        e.g. 80 beats per minute
+    Blood Pressure: systolic blood pressure / diastolic blood pressure
+        e.g. 120/80
+
+'''
+def create_tables(db):
+    print("Creating tables . . . ")
+    # Checks if table exists before creation
+    db.execute('create table if not exists Data (PatientID TEXT, Timestamp TEXT, Type TEXT, Value TEXT)')
+    db.execute('create table if not exists Alerts (PatientID TEXT, Timestamp TEXT, Type TEXT, ThresholdLow TEXT, ThresholdHigh TEXT, Value TEXT)')
+
+'''
+Description:
+    add_data_item adds one type (bp, o, p) item to the data database
+
+inputs:
+    patientID: The patient's identification code
+    timestamp: Time recorded
+    entrytype: Either Oxygen 'O', Pulse 'P', or Blood Pressure 'BP'
+    value:     The type's value to be stored
+'''
+def add_data_item(db, patientID, timestamp, entrytype, value):
+    if value is None:
+        return
+
+    db.execute("INSERT INTO Data (PatientID, Timestamp, Type, Value) \
+               VALUES (?,?,?,?)",(patientID, timestamp, entrytype, value) )
+    db.commit()
+    print("Data item added successfully")
+
+'''
+Description:
+    add_all_types_items adds up to all three items to the data database;
+    when calling, use parameterized function syntax
+
+inputs:
+    patientID:      The patient's identification code
+    timestamp:      Time recorded
+    oxygenValue:    Oxygen level, can be left out
+    pulseValue:     Pulse value, can be left out
+    bpValue:        Blood Pressure level, can be left out
+'''
+def add_all_types_items(db, patientID, timestamp, oxygenValue=None, pulseValue=None, bpValue=None):
+    add_data_item(db, patientID, timestamp, 'O', oxygenValue)
+    add_data_item(db, patientID, timestamp, 'P', pulseValue)
+    add_data_item(db, patientID, timestamp, 'BP', bpValue)
+
+'''
+Description:
+    add_alerts_item adds an alert item to the alert database
+
+inputs:
+    patientID:      The patient's identification code
+    timestamp:      Time recorded
+    entrytype:      Either Oxygen 'O', Pulse 'P', or Blood Pressure 'BP'
+    threshold_low:  Lower threshold value for type entrytype
+    threshold_high: Higher threshold value for type entrytype
+    value:          actual recorded value
+'''
+def add_alerts_item(db, patientID, timestamp, entrytype, threshold_low, threshold_high, value):
+    db.execute("INSERT INTO Alerts (PatientID, Timestamp, Type, ThresholdLow, ThresholdHigh, Value) \
+               VALUES (?,?,?,?,?,?)",(patientID, timestamp, entrytype, threshold_low, threshold_high, value) )
+    db.commit()
+    print("Alerts item added successfully")
```

Comment:
Same issue here. The alerts module shouldn't have anything to do with timestamps and db instances. That should be contained within the database module itself.